### DISCUSSION TITLE
Improved .bootenv

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#196] [General]
Running boot/.bootenv everytime one logs in can take awhile - longer than it needs to.

My suggestion is that once the boot/.bootenv is created that it run and create a file that contains all of the settings in one and without all the extra processing.  Perhaps a boot/.envuse file.

That boot/.envuse can then be updated if/when things change.

**Comment by MikeFultonDev** [comment]
This is getting a LOT better as we eliminate the need for most tools to do any 'work'. @lbdyck would love to get your input on the very latest drop (post Sep 22) on this...
